src/Maestro/ReleasePipelineRunner/Program.cs

```diff
@@ -48,6 +50,18 @@ private static void Main()
                                         options.Tokens.Add(token.GetValue<string>("Account"), token.GetValue<string>("Token"));
                                     }
                                 });
+                            services.AddGitHubTokenProvider();
+                            services.Configure<GitHubTokenProviderOptions>(
+                                (options, provider) =>
+                                {
+                                    var config = provider.GetRequiredService<IConfigurationRoot>();
+                                    IConfigurationSection section = config.GetSection("GitHub");
+                                    section.Bind(options);
+                                    options.ApplicationName = "Maestro";
```

Comment:
Nit: Maestro++

Reply:
This is a copy of what SubscriptionActorService has.